src/alerting/utils/activity.ts

```diff
@@ -7,20 +7,24 @@ import {event} from 'src/cloud/utils/reporting'
 import {MONITORING_BUCKET} from 'src/alerting/constants'
 
 // Types
-import {CancelBox, StatusRow} from 'src/types'
+import {CancelBox, CheckIDsMap, StatusRow} from 'src/types'
 import {RunQueryResult} from 'src/shared/apis/query'
 import {LoadRowsOptions, Row} from 'src/types'
 
 export const runAlertsActivityQuery = (
   orgID: string,
+  checkIDs: CheckIDsMap,
   {offset, limit, since}: LoadRowsOptions
 ): CancelBox<StatusRow[]> => {
   const start = since ? Math.round(since / 1000) : '-1d'
 
+  // Empty checkIDs set will result in 0 records being fetched by the following query
+  const checks = Object.keys(checkIDs).join('", "')
```

Comment:
👍 for pulling this out into it's own line. It being inline with the query made my dumb brain hurt.

Reply:
thank you for your review, sir!